zone/gm_commands/summon.cpp

```diff
@@ -7,82 +7,79 @@ extern WorldServer worldserver;
 
 void command_summon(Client *c, const Seperator *sep)
 {
-	Mob *t;
+	int arguments = sep->argnum;
+	if (!arguments && !c->GetTarget()) {
```

Comment:
May not want to require a target to get the help menu.

You could have a target and only type `#summon` but then not get help menu

Reply:
Well, if you have a target and use #summon, it will summon your target.

The idea is you would only ever need the help menu if you don't currently have a target, otherwise I'd have to add a parameter to tell it when to summon when you have a target and when not to summon when you have a target.

Reply:
That's fair